docs/src/content/faq.mdx

```diff
@@ -48,27 +81,275 @@ After receiving the NFT, follow these steps to set up the Icebreaker platform:
 
 For detailed instructions on each step, follow the links above.
 
-## How do I get access to the private channel with Icebreaker documentation?
+### What endpoints do I need to open in my firewall for the Icebreaker node?
 
-After receiving your Icebreaker NFT:
+To function properly, your Icebreaker node requires the following ports to be open in your firewall:
 
-- Notify the Blockfrost team that you've received your NFT
-- A team member will add you to the private Icebreakers channel
-- Access the comprehensive documentation in the announcement channel
-- Follow the setup instructions provided there.
+- `3000/tcp` (or custom port): For incoming API requests from Blockfrost
+- `8081/tcp`: Prometheus metrics endpoint for monitoring (optional but recommended)
 
-If you've received your NFT but don't have channel access yet, post a message on [Discord](https://discord.com/invite/inputoutput) in the `#ask-blockfrost` channel for assistance.
+Ensure these ports allow incoming connections from the Blockfrost server IPs documented [here](/usage/network#firewall).
 
-## Do I need confirmation after receiving the NFT to proceed?
+You can verify port accessibility using:
+```bash
+sudo ufw allow 3000/tcp
+sudo ufw allow 8081/tcp
+```
 
-No, you don't need additional confirmation after receiving the NFT.
-Once the NFT is in your wallet, you can proceed with installation and setup.
-However, you'll need access to the private Icebreakers channel for complete documentation, which is granted after notifying the team that you've received your NFT.
+### How do I build `blockfrost-platform` from source?
 
-## Can I transfer my Icebreaker NFT to another wallet address?
+Refer to the [installation instructions](/installation/source) for detailed steps on building from source.
+This is often recommended over using pre-built installers if you encounter issues with the provided binaries.
 
-Yes, you can transfer your Icebreaker NFT to another wallet address by following these steps:
+### Do I need a static public IP address for my Icebreaker node?
+
+Yes, a static public IP is strongly recommended for reliably running an Icebreaker node.
+
+The Blockfrost platform currently identifies and monitors nodes based on their consistent public IP. Without a static IP, you risk connection issues and flapping.
+
+A future update will allow dynamic IPs by having the node proactively report its address. See [this GitHub issue](https://github.com/blockfrost/blockfrost-platform/issues/178) for details on that roadmap.
+
+In the interim, using a static IP is best. If not possible, a dynamic DNS service or VPN may help provide a stable hostname to IP mapping.
+
+### What should I do if my public IP address changes frequently?
```

Comment:
Similar: It would be nice to mention that we've developed a reverse-connection method using WebSockets. It's not in production yet, but will make it easy to run behind a firewall, double-nat, cgnat and without a static ip.

Reply:
I mentioned it.